x/incentive/client/cli/query.go

```diff
@@ -86,62 +94,62 @@ func queryRewardsCmd(queryRoute string, cdc *codec.Codec) *cobra.Command {
 			switch strings.ToLower(strType) {
 			case typeHard:
 				params := types.NewQueryRewardsParams(page, limit, owner, boolUnsynced)
-				claims, err := executeHardRewardsQuery(queryRoute, cdc, cliCtx, params)
+				claims, err := executeHardRewardsQuery(cliCtx, params)
 				if err != nil {
 					return err
 				}
-				return cliCtx.PrintOutput(claims)
+				return cliCtx.PrintObjectLegacy(claims)
 			case typeUSDXMinting:
 				params := types.NewQueryRewardsParams(page, limit, owner, boolUnsynced)
-				claims, err := executeUSDXMintingRewardsQuery(queryRoute, cdc, cliCtx, params)
+				claims, err := executeUSDXMintingRewardsQuery(cliCtx, params)
 				if err != nil {
 					return err
 				}
-				return cliCtx.PrintOutput(claims)
+				return cliCtx.PrintObjectLegacy(claims)
 			case typeDelegator:
 				params := types.NewQueryRewardsParams(page, limit, owner, boolUnsynced)
-				claims, err := executeDelegatorRewardsQuery(queryRoute, cdc, cliCtx, params)
+				claims, err := executeDelegatorRewardsQuery(cliCtx, params)
 				if err != nil {
 					return err
 				}
-				return cliCtx.PrintOutput(claims)
+				return cliCtx.PrintObjectLegacy(claims)
 			case typeSwap:
 				params := types.NewQueryRewardsParams(page, limit, owner, boolUnsynced)
-				claims, err := executeSwapRewardsQuery(queryRoute, cdc, cliCtx, params)
+				claims, err := executeSwapRewardsQuery(cliCtx, params)
 				if err != nil {
 					return err
 				}
-				return cliCtx.PrintOutput(claims)
+				return cliCtx.PrintObjectLegacy(claims)
 			default:
 				params := types.NewQueryRewardsParams(page, limit, owner, boolUnsynced)
 
-				hardClaims, err := executeHardRewardsQuery(queryRoute, cdc, cliCtx, params)
+				hardClaims, err := executeHardRewardsQuery(cliCtx, params)
 				if err != nil {
 					return err
 				}
-				usdxMintingClaims, err := executeUSDXMintingRewardsQuery(queryRoute, cdc, cliCtx, params)
+				usdxMintingClaims, err := executeUSDXMintingRewardsQuery(cliCtx, params)
 				if err != nil {
 					return err
 				}
-				delegatorClaims, err := executeDelegatorRewardsQuery(queryRoute, cdc, cliCtx, params)
+				delegatorClaims, err := executeDelegatorRewardsQuery(cliCtx, params)
 				if err != nil {
 					return err
 				}
-				swapClaims, err := executeSwapRewardsQuery(queryRoute, cdc, cliCtx, params)
+				swapClaims, err := executeSwapRewardsQuery(cliCtx, params)
 				if err != nil {
 					return err
 				}
 				if len(hardClaims) > 0 {
-					cliCtx.PrintOutput(hardClaims)
+					cliCtx.PrintObjectLegacy(hardClaims)
```

Comment:
Would return for these since `PrintObjectLegacy` also return errors

```suggestion
					return cliCtx.PrintObjectLegacy(hardClaims)
```

Reply:
Misunderstood this, we can just handle handle error but not return since we're printing multiple things here.

Reply:
right, i'll update to return only if we get an error